production/elasticsearch.yml

```diff
@@ -0,0 +1,105 @@
+#
+# Copyright 2017 The Jaeger Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+
+# This template is based on the Elasticsearch staging example: 
+# https://github.com/kubernetes/examples/tree/master/staging/elasticsearch
+# It's not intended for production purposes, but it's simple enough for 
+# an example on our Jaeger production deployment.
+
+apiVersion: v1
+kind: Template
+labels:
+  app: jaeger
+  template: jaeger-elasticsearch-template
+  jaeger-infra: template-elasticsearch
+metadata:
+  name: jaeger-elasticsearch-template
+  annotations:
+    description: Elasticsearch storage for Jaeger
+    iconClass: icon-go-gopher
+    openshift.io/display-name: Jaeger's Elasticsearch
+    tags: instant-app,tracing,opentracing,jaeger,elasticsearch
+  labels:
+    app: jaeger
+    name: jaeger-infra
+    jaeger-infra: template-production-elasticsearch
+objects:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: elasticsearch
```

Comment:
Should these be qualified by jaeger, e.g. `jaeger-elasticsearch`, so that if these templates are used then it is only intended for use with jaeger? and to avoid potential name conflicts with other es installations.

Reply:
Possibly not an issue if installed in separate namespace as described in the instructions - but something to consider.

Reply:
Not quite sure. I see the `elasticsearch` and `cassandra` templates now as something "external" to us, placed here only for convenience. I'd try to keep those as close to the upstream as possible, but if your (and Pavol's) views are different, I'm fine in renaming this.

Reply:
No problem - not a big issue.